interface-definitions/service-event-handler.xml.in

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <interfaceDefinition>
-  <node name="system">
+  <node name="service">
     <children>
-      <tagNode name="event-handler" owner="${vyos_conf_scripts_dir}/system_event_handler.py">
+      <tagNode name="event-handler" owner="${vyos_conf_scripts_dir}/service_event_handler.py">
```

Comment:
I still think it should be `set service event-handled event SomeEvent pattern ...`.

Here's why: suppose we want to add global event-handler options in the future. If the event handler has its own subtree under `service`, it's trivial. If it's a bare tag node, however, those global options will have to go outside that tag node, and we'll either end up with `set service event-handler-options` (which is ugly) or have to write a migration script.

A few more lines of XML now can save us a lot of effort later in that regard.

Reply:
@dmbaturin Done
```
set service event-handler event first filter pattern '.*ssh2.*'
set service event-handler event first script arguments '192.0.2.5'
set service event-handler event first script environment interface value 'eth0'
set service event-handler event first script path '/config/scripts/hello.sh'

```